test/connectivity/modules/NetworkController/LinuxTrafficControlController.cs

```diff
@@ -102,16 +102,22 @@ async Task<bool> AddDropRule(CancellationToken cs)
 
                 // Adding rules to filter packages by iotHub IP
                 // Details about how the rules work https://wiki.archlinux.org/index.php/Advanced_traffic_control
+                Log.LogInformation($"Executing: {LinuxTrafficControllerHelper.GetRootRule(this.networkInterfaceName)}");
```

Comment:
These logs were originally just for testing, but they seem helpful, so I kept them in